srcpkgs/reuse/template

```diff
@@ -1,18 +1,20 @@
 # Template file for 'reuse'
 pkgname=reuse
-version=0.13.0
-revision=2
+version=1.0.0
+revision=1
 build_style=python3-module
+# These tests pass on local machine but don't pass in CI.
 make_check_args="--deselect tests/test_lint.py::test_lint_read_errors
+ --deselect tests/test_main_addheader.py::test_addheader_to_read_only_file_does_not_traceback
  --deselect tests/test_report.py::test_generate_project_report_read_error"
-hostmakedepends="python3-setuptools python3-setuptools_scm gettext"
-depends="python3-binaryornot python3-boolean.py python3-debian python3-Jinja2
- python3-license-expression python3-requests"
-checkdepends="python3-pytest $depends"
+hostmakedepends="gettext python3-setuptools python3-setuptools_scm"
+depends="python3-Jinja2 python3-binaryornot python3-boolean.py python3-debian
+ python3-license-expression python3-requests python3-setuptools"
+checkdepends="$depends git mercurial python3-pytest"
```

Comment:
there is no need to move stuff around, it only creates an unreadable diff, could you fix that?